services/QuillLMS/app/controllers/statuses_controller.rb

```diff
@@ -34,6 +34,13 @@ def redis_queue
     render plain: 'OK'
   end
 
+  def sidekiq_queue_latency
+    queues = Sidekiq::Queue.all
+    latency_hash = queues.map { |q| [q.name, q.latency] }.to_h
```

Comment:
Some code golf:
`latency_hash = queues.to_h { |q| [q.name, q.latency] }`

Reply:
Oh, hey, I didn't know you could do that with `to_h`.  Today I learned!